### ISSUE TITLE
Issue with name suffix, like "Jr."

### ISSUE THREAD

**Issue by glorious1** [dsifford/astrocite#4]
**ABT Version:** 4.9.8

**PHP Version:** 7.2.11

**Theme:** GeneratePress

**Plugin List:**
- Plugin 1
- Plugin 2

**Browser:** Firefox 62.0.3

**Expected behavior:**

**Actual behavior:**

**Console messages:**
```js
// Paste messages here

```
A name suffix such as Jr. doesn't work for me.  In Zotero, using the same csl that I am using in ABT, I can enter a name as:
Cobb, F. W., Jr. and it is styled/formatted as the csl intends:
Cobb FW Jr

But in ABT, the same name appears in the bibliography as
Cobb J FW

Perhaps related, if the first name is spelled out:
Cobb, Fields W., Jr. it should still be rendered (and is by Zotero) as Cobb FW Jr, but ABT renders it as
Cobb J Fields W

By the way, still loving the plugin, and I understand you have a lot going on and may not be able to address things for a while.

**Comment by dsifford** [comment]
@glorious1 how did these names get entered into Zotero? Did you do it by hand?

It appears that the form for the name suffixes are just being entered incorrectly. 

Where you have `Cobb, F. W., Jr.` and `Cobb, Fields W., Jr.`, it should be `Cobb, Jr., F.W.` and `Cobb, Jr., Fields W.` respectively.

Did zotero place those names in that format?

**Comment by dsifford** [comment]
I just downloaded zotero and exported [this paper](http://dx.doi.org/10.1139/b72-247) to RIS and the format that it generated was  `Cobb Jr., F. W.`, which is valid and parses correctly. So I'm gonna mark this as invalid unless anything new comes up.

Reopen this if I missed something.

**Comment by glorious1** [comment]
Actually, the Zotero developers say Jr., etc. should be entered after the given name, preceded by a comma:
Lastname: Smith
Firstname: Adam, Jr.
Here is a recent response on that: 
https://forums.zotero.org/discussion/75479/generation-designations-jr-iii-etc

Testing again today, I find both methods (after last name and after first name) are resulting in a mostly logical presentation in ABT.  Not sure if that is related to changes in ABT or my style.

Putting it after the last name means, if the text citation use names, it will be part in the citations, which is not usually done.  

Bottom line, it doesn't seem to be an issue any more!

**Comment by dsifford** [comment]
They are mistaken. I'll clarify that with them. Thanks for the info.
